RadixWallet/Features/CardCarousel/CardCarousel+View.swift

```diff
@@ -0,0 +1,194 @@
+import Foundation
+import SwiftUI
+
+// MARK: - CardCarousel.View
+extension CardCarousel {
+	public struct View: SwiftUI.View {
+		public let store: StoreOf<CardCarousel>
+
+		private let margin: CGFloat = .medium1
+		private let spacing: CGFloat = .small1 * 0.5
+		@ScaledMetric private var height: CGFloat = 105
+		@SwiftUI.State private var selectedCardIndex = 0
+
+		public var body: some SwiftUI.View {
+			WithPerceptionTracking {
+				VStack(spacing: 8) {
+					GeometryReader { _ in
+						coreView
+					}
+					.frame(height: store.cards.isEmpty ? 0 : height)
```

Comment:
could we rather not add the view if there are no cards?

Reply:
given the logic to determine if there are (or not) cards is inside the view's reducer, we can't